tasks/systemd.yml

```diff
@@ -6,3 +6,9 @@
     owner: "root"
     mode: "u=rwx,g=r,o=r"
   notify: "restart node_exporter"
+
+- name: "Make sure node exporter service is running and auto-starting"
+  service:
+    name: "node_exporter"
+    state: "started"
```

Comment:
How about reloaded instead of started? I mean, previous task is setting up service which means theoretically it might happen that it update its configuration so maybe reloading is better than just ensuring that it is running...

Reply:
that is a good point, for that purpose ansible has `notify` mechanism, see also above.

what happens there, if the task with `notify` ends up in "changed" state, ansible will set a flag and call corresponding notification handler which is usually service `restart` or `reload`. I think that covers the case that you have suggested

And what I wanted to do - make sure that service is added to system auto-start list is also covered, so we should be good with that role :)